src-js/app.js

```diff
@@ -1,19 +1,29 @@
 import * as Cookie from 'js-cookie';
-import * as search from './instantsearch.js';
-import * as lang from './lang.js'
+import * as highlightjsBadge from 'highlightjs-badge';
+
+
+// site files
 import OptimizelyClient from './optimizely.js';
 import AnalyticsClient from "./analytics.js";
+import * as search from './instantsearch.js';
+import * as lang from './lang.js'
+import './user.js';
+import './sidebar.js';
+import './nav.js';
+import './main.js';
+
+// imported but not used just so webpack picks it up and add it to the `app.bundle.js`
+import * as hljsBadge from 'highlightjs-badge';
+import './highlightJSBadge.js'
```

Comment:
Can we move this to the `experiments/` directory now?

Reply:
Done.